shell/browser/native_window.cc

```diff
@@ -693,6 +697,17 @@ void NativeWindow::NotifyWindowMessage(UINT message,
 #endif
 
 int NativeWindow::NonClientHitTest(const gfx::Point& point) {
+#if !BUILDFLAG(IS_MAC)
+  // We need to ensure we account for resizing borders on Windows and Linux.
+  if ((!has_frame() || has_client_frame()) && IsResizable()) {
+    auto* frame =
+        static_cast<FramelessView*>(widget()->non_client_view()->frame_view());
+    int border_hit = frame->ResizingBorderHitTest(point);
```

Comment:
This is also checked in `NativeWindowViews::ShouldDescendIntoChildForEventHandling`, we should either remove the same check in that method, or just fix the issue in that method instead.

Reply:
@zcbenz what do you mean by fix the issue in that method instead? Even when the check there correctly returns false for points on the frame, this issue still occurs (i checked that initially)

Reply:
I thought reordering the check in `ShouldDescendIntoChildForEventHandling` would fix the issue, since it does not, I think we can just remove the duplicate check there.

Reply:
just made the change - i think that should do it!